test_communication/test/test_secure_publisher_subscriber.py.in

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler
+from launch.launcher import DefaultLauncher
+
+
+def test_secure_publisher_subscriber():
+    ld = LaunchDescriptor()
+    publisher_cmd = [
+        '@TEST_PUBLISHER_EXECUTABLE@', '@TEST_MESSAGE_TYPE@'
+    ]
+    subscriber_cmd = [
+        '@TEST_SUBSCRIBER_EXECUTABLE@', '@TEST_MESSAGE_TYPE@',
+        '@SUBSCRIBER_SHOULD_TIMEOUT@'
+    ]
+
+    # TODO (mikaelarguedas) Do we want to test any of this across client libraries??
```

Comment:
this is testing only for rclcpp and I assumed it was enough because the goal here is to test the underlying rcl feature. I kept the comment as is in case anybody think otherwise and I can enable multi rcl tests if it's deemed necessary.